modules/serverless-install-kafka-odc.adoc

```diff
@@ -0,0 +1,34 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-install-kafka-odc_{context}"]
+= Installing Apache Kafka components using the web console
+
+Cluster administrators can enable the use of Apache Kafka functionality in a {ServerlessProductName} deployment by instantiating the KnativeKafka custom resource definition provided by the *Knative Kafka* {ServerlessOperatorName} API.
+
+.Prerequisites
+* You must have access to an AMQ Streams cluster.
+* You must have installed the {ServerlessOperatorName} and Knative Eventing.
+* You must have cluster administrator permissions on the {product-title} cluster.
+* You must have logged in to the web console.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Operators* → *Installed Operators*.
+. Check that the *Project* dropdown at the top of the page is set to *Project: knative-eventing*.
+. Click *Knative Kafka* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Kafka* tab.
++
+image::knative-kafka-api.png[OpenShift Serverless Operator provided APIs]
```

Comment:
Decided to actually remove this one since it doesn't really add much, and the Operator version will be outdated quickly so it'll just be a maintenance headache later.